src/internal/utils.ts

```diff
@@ -1,29 +1,29 @@
-import { GlobalVars } from "./globalVars";
+import { GlobalVars } from './globalVars';
 
 // This will return a reg expression a given url
 function generateRegExpFromUrl(url: string): string {
-  let urlRegExpPart = "^";
-  const urlParts = url.split(".");
-  for (let j = 0; j < urlParts.length; j++) {
-    urlRegExpPart += (j > 0 ? "[.]" : "") + urlParts[j].replace("*", "[^/^.]+");
-  }
-  urlRegExpPart += "$";
-  return urlRegExpPart;
+    let urlRegExpPart = '^';
+    const urlParts = url.split('.');
+    for (let j = 0; j < urlParts.length; j++) {
+        urlRegExpPart += (j > 0 ? '[.]' : '') + urlParts[j].replace('*', '[^/^.]+');
+    }
+    urlRegExpPart += '$';
+    return urlRegExpPart;
 }
 
 // This will return a reg expression for list of url
 export function generateRegExpFromUrls(urls: string[]): RegExp {
-  let urlRegExp = "";
-  for (let i = 0; i < urls.length; i++) {
-    urlRegExp += (i === 0 ? "" : "|") + generateRegExpFromUrl(urls[i]);
-  }
-  return new RegExp(urlRegExp);
+    let urlRegExp = '';
+    for (let i = 0; i < urls.length; i++) {
+        urlRegExp += (i === 0 ? '' : '|') + generateRegExpFromUrl(urls[i]);
+    }
+    return new RegExp(urlRegExp);
 }
 
 export function getGenericOnCompleteHandler(errorMessage?: string): (success: boolean, reason?: string) => void {
-  return (success: boolean, reason: string) => {
-    if (!success) {
-      throw new Error(errorMessage ? errorMessage : reason);
-    }
-  };
-}
\ No newline at end of file
+    return (success: boolean, reason: string) => {
```

Comment:
Any chance we can stick with 2 spaces of indentation? That's what our other code bases use and I think it's good to be consistent.

Reply:
updated with 2 space indentation